### DISCUSSION TITLE
feat: provide a switch for updating pop ups

### DISCUSSION THREAD

**Discussion by Jaylenwa** [open-webui/open-webui#6351] [General]
Can you provide a switch to control whether this box pops up? When using a new device, it will always pop up. I think it is unnecessary and may affect the experience of some users who do not need to update.
This is just my personal feeling. Thank you very much.

![image](https://github.com/user-attachments/assets/851d70a6-83dd-4319-b3f1-2a6757c2cf2d)

**Comment by tjbck** [comment]
Addressed in dev.
